crates/hyperswitch_connectors/src/connectors/novalnet/transformers.rs

```diff
@@ -183,26 +184,29 @@ impl TryFrom<&NovalnetRouterData<&PaymentsAuthorizeRouterData>> for NovalnetPaym
         };
 
         let billing = NovalnetPaymentsRequestBilling {
-            house_no: item.router_data.get_billing_line1()?,
-            street: item.router_data.get_billing_line2()?,
-            city: Secret::new(item.router_data.get_billing_city()?),
-            zip: item.router_data.get_billing_zip()?,
-            country_code: item.router_data.get_billing_country()?,
+            house_no: item.router_data.get_billing_line1().ok(),
```

Comment:
```suggestion
            house_no: item.router_data.get_optional_billing_line1(),
```
Can you use optional utils function in all other places ?